news/5502-python_site_packages_path-can-be-specified-in-the-build-section

```diff
@@ -0,0 +1,19 @@
+### Enhancements
+
+* The location of the site-packages directory can be specified in `python_site_packages_path` in the build section of a recipe (#5502).
```

Comment:
```suggestion
* Add option for the location of the site-packages directory to be specified via `python_site_packages_path` in the build section of a recipe for any packages named `python`. (#5502)
```

Reply:
Let's mention CEP-17 as well, like in https://github.com/conda/conda/pull/14256/files#diff-34b92149e59e8ccca3335eebd8e8be442c5683b59ead38a89f9a47cd460a3aaeR3

Reply:
Like so?

```suggestion
* Add  support for CEP-17 that allows specifying the location of the site-packages directory with the `python_site_packages_path` build option for any packages named `python`. (#5502)
```